install.sh

```diff
@@ -0,0 +1,537 @@
+#!/bin/sh
+set -eu
+
+# Coder's automatic install script.
+# See https://github.com/coder/coder#installing-coder
+
+usage() {
+	arg0="$0"
+	if [ "$0" = sh ]; then
+		arg0="curl -fsSL https://coder.com/install.sh | sh -s --"
+	else
+		not_curl_usage="The latest script is available at https://coder.com/install.sh
+"
+	fi
+
+	cath <<EOF
+Installs Coder.
+It tries to use the system package manager if possible.
+After successful installation it explains how to start Coder.
+
+Pass in user@host to install Coder on user@host over ssh.
+The remote host must have internet access.
+${not_curl_usage-}
+Usage:
+
+  $arg0 [--dry-run] [--version X.X.X] [--edge] [--method detect] \
+        [--prefix ~/.local] [--rsh ssh] [user@host]
+
+  --dry-run
+      Echo the commands for the install process without running them.
+
+  --version X.X.X
+      Install a specific version instead of the latest.
+
+  --edge
+      Install the latest edge version instead of the latest stable version.
+
+  --method [detect | standalone]
+      Choose the installation method. Defaults to detect.
+      - detect detects the system package manager and tries to use it.
+        Full reference on the process is further below.
+      - standalone installs a standalone release archive into ~/.local
+        Add ~/.local/bin to your \$PATH to use it.
+
+  --prefix <dir>
+      Sets the prefix used by standalone release archives. Defaults to ~/.local
+      The release is unarchived into ~/.local/lib/coder-X.X.X
+      and the binary symlinked into ~/.local/bin/coder
+      To install system wide pass ---prefix=/usr/local
```

Comment:
@kylecarbs actually, what do you think of defaulting to `/usr/local/bin`? code-server defaults to the home-directory one, but it's often not in PATH. [helm's install script](https://github.com/helm/helm/blob/main/scripts/get-helm-3#L25) does this, so I wouldn't consider it too .

Reply:
done in 6879752d98a39007591102667a0f1781d7738e6e